src/GrumPHP/Task/Phpcs.php

```diff
@@ -84,4 +94,34 @@ public function run(ContextInterface $context)
 
         return TaskResult::createPassed($this, $context);
     }
+
+    /**
+     * @param array $config
+     *
+     * @return null|string
+     */
+    protected function getWhitelistPathPattern(array $config)
+    {
+        /** @var string|null $whitelistPathPattern */
+        $whitelistPathPattern = $config['whitelist_path_pattern'];
+        if (!$whitelistPathPattern) {
+            return null;
+        }
+
+        $whitelistPathPattern = $this->escapePatternDirectorySeparator($whitelistPathPattern);
+        $whitelistPathPattern = '/'.$whitelistPathPattern.'.(%s)$/';
```

Comment:
That's strange.  Did you also try with single quotes?
It is also possible that you also need to add the delimiters to the yaml file to make sure the glob filter is not applied or you could just use the glob filter instead with: `src/*`